x/fswap/keeper/expected_keepers.go

```diff
@@ -0,0 +1,20 @@
+package keeper
+
+import (
+	sdk "github.com/Finschia/finschia-sdk/types"
+)
+
+type (
+	AccountKeeper interface {
+		GetModuleAddress(name string) sdk.AccAddress
+	}
+
+	BankKeeper interface {
```

Comment:
We should consider using `IsSendEnabledCoins()`.

Reply:
Are you saying that we need to call `IsSendEnabledCoins()` before calling `SendCoinsFromAccountToModule` and `SendCoinsFromModuleToAccount`?
If so, Is it because that two method doesn't check `IsSendEnabledCoins()`?

Reply:
You don't need to call `IsSendEnabledCoins()` before every single call. Just make sure the denom of user input is enabled for sending (e.g. on message server).

Reply:
Okay, I'll make a call `IsSendEnabledCoins()` for `Swap` and `SwapAll` txs.